scripts/add-icon.js

```diff
@@ -0,0 +1,71 @@
+import fs from 'node:fs/promises';
+import inquirer from 'inquirer';
+import chalkPipe from 'chalk-pipe';
```

Comment:
Why are you using `chalk-pipe` instead of `chalk` directly? `chalk` is a direct dependence of `inquirer`, could we avoid the addition of `chalk-pipe` to our development dependencies?

Reply:
`chalk-pipe` can automatically recognize style strings. It is designed for those who need to use mutable styles. They can switch to another color scheme with a simple string: like from `blue.bold.underline` to `bg#ff99cc.yellow`.

I think we might need more features. But in the current case, the `chalk` is enough to use.

Will use `chalk` instead.